### PR TITLE
[Benchmark/Stream] Add getting started docs for elastic-package stream command

### PR DESCRIPTION
**Pull request by ruflin**
The goal of these docs is that any engineer can quickly ingest some sample data. It is specially targeted on engineers working on Kibana and need sample data.

### CODE REVIEW

File: docs/howto/sample_data.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+cd packages/nginx
+```
+
+Inside the directory, run the following command with the `elastic-package` stack setup.
+
+```
+elastic-package benchmark stream -v
+```
+
+If I run I run Kibana from source, I run the following command:
```
Comment (gbamparop) [inline docs/howto/sample_data.md:49]: If Kibana starts with a basepath (default behavior with `yarn start`) or a `server.basePath` is set in the config, the `ELASTIC_PACKAGE_KIBANA_HOST` variable should be updated to include the path. If the env variable is left as is the issue is not really obvious from the `elastic-package` output.

Comment (ruflin) [reply docs/howto/sample_data.md:49]: Good point. I'm so used to start Kibana with --no-base-path I missed it. I need to test if it works when the base path is added to the env variable or @gbamparop did you already try it?

File: docs/howto/sample_data.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+```
+ELASTIC_PACKAGE_ELASTICSEARCH_HOST=http://localhost:9200
+ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME=elastic
+ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD=changeme
+ELASTIC_PACKAGE_KIBANA_HOST=http://localhost:5601
+#ELASTIC_PACKAGE_CA_CERT
+```
+
+The same can be use to connect to your Elastic Cloud cluster. Unfortunately elastic-package currently does not support an API key to be used.
```
[the remark this answers is not in the document]
Comment (ruflin) [reply docs/howto/sample_data.md:25]: I thought we had one as we discussed it previously but couldn't find it. @aspacca do you remember?

If it is missing, will create it.

File: docs/howto/sample_data.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+As soon as elastic-package is set up, you need a running Elastic. The simplest thing to set it up with Docker is running `elastic-package stack up --version=8.13.0-SNAPSHOT -v`. This will start a snapshot build of an Elastic Stack cluster on your machine and you can open Kibana under `https://localhost:5601`.
+
+If you have your own Cloud setup or run Kibana from source, you can set the environment variables accordingly. The following is a sample assuming you are running Kibana and Elasticsearch from source: 
+
+```
+ELASTIC_PACKAGE_ELASTICSEARCH_HOST=http://localhost:9200
+ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME=elastic
+ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD=changeme
+ELASTIC_PACKAGE_KIBANA_HOST=http://localhost:5601
```
[the remark this answers is not in the document]
Comment (ruflin) [reply docs/howto/sample_data.md:21]: See discussion above with @gbamparop Need to test and validate and will add a section around it.

File: docs/howto/sample_data.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (aspacca) [reply docs/howto/sample_data.md:25]: @ruflin @jsoriano the issue still needs to be created

**Review by aspacca — APPROVED** [review]

File: docs/howto/sample_data.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ruflin) [reply docs/howto/sample_data.md:25]: I created https://github.com/elastic/elastic-package/issues/1633

File: docs/howto/sample_data.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ruflin) [reply docs/howto/sample_data.md:21]: Added an example below.

File: docs/howto/sample_data.md (hunk 1 above)
Comment (ruflin) [reply docs/howto/sample_data.md]: Tried it out and added an example.

**Comment by ruflin** [comment]
Follow up PR to generate list: https://github.com/elastic/elastic-package/pull/1634

**Review by jsoriano — APPROVED** [review]
Thanks!

**Pull request merged**